src/test/java/org/apache/drill_web_test_framework/web_ui/pages/OptionsPage.java

```diff
@@ -0,0 +1,271 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill_web_test_framework.web_ui.pages;
+
+import org.apache.drill_web_test_framework.properties.PropertiesConst;
+import org.apache.drill_web_test_framework.web_ui.WebBrowser;
+import org.openqa.selenium.By;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+
+import java.time.Duration;
+import java.util.List;
+
+public class OptionsPage extends BasePage {
+  private FluentWait<WebDriver> fluentWait = new FluentWait<>(WebBrowser.getDriver())
+      .withTimeout(Duration.ofSeconds(PropertiesConst.DEFAULT_TIMEOUT))
+      .pollingEvery(Duration.ofMillis(200))
+      .ignoring(StaleElementReferenceException.class);
+
+  @FindBy(id = "searchBox")
+  private WebElement searchField;
+
+  @FindBy(css = "button[title='Clear search']")
+  private WebElement clearSearchButton;
+
+  @FindBy(id = "optionsTbl")
+  private WebElement optionTable;
+
+  @FindBy(xpath = "//button[text()='planner']")
+  private WebElement plannerFilter;
+
+  @FindBy(xpath = "//button[text()='store']")
+  private WebElement storeFilter;
+
+  @FindBy(xpath = "//button[text()='parquet']")
+  private WebElement parquetFilter;
+
+  @FindBy(xpath = "//button[text()='hashagg']")
+  private WebElement hashaggFilter;
+
+  @FindBy(xpath = "//button[text()='hashjoin']")
+  private WebElement hashjoinFilter;
```

Comment:
Quick list can change in future, better to parametrize it's elements name. Maybe, use an enum?

Reply:
I think it is better leave it like this